backend/stories/schemas.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
+
+from pydantic import BaseModel
+from typing import List
+from . import models
+from enum import Enum
+from users import models as user_models
+
+class Sex(str, Enum):
+    female = "female"
+    male = "male"
+    other = "other"
+
+class MedicalSituation(str, Enum):
+    sick = "sick"
+    not_sick = "not sick"
+    recovered = "recovered"
+
+class TestSituation(str, Enum):
+    positive = "positive"
+    negative = "negative"
+    not_tested = "not tested"
+
+class CreateStory(BaseModel):
```

Comment:
The nomeclature is inconsistent with what we had on the `users` schema - we should either change it over there or over here!

Reply:
Your are right. I change Story to be consistent